libsolidity/parsing/Token.cpp

```diff
@@ -160,6 +162,7 @@ tuple<Token::Value, unsigned int, unsigned int> Token::fromIdentifierOrKeyword(s
 					m % 8 == 0 &&
 					n % 8 == 0
 				) {
+					solAssert(n != -1, "Invalid type N in fixed command. Should not reach here.");
```

Comment:
alright. I think I see what you're trying to say. I'll add it into the conditions.